app/controllers/api/v1/api_controller.rb

```diff
@@ -7,6 +7,7 @@ class ApiController < ActionController::API
       after_action :verify_authorized
 
       before_action :authenticate_devise_api_token!
+      before_action :set_paper_trail_whodunnit
```

Comment:
WDYF about also adding it  to `ApplicationController`?

Reply:
The papertrail has an attribute `whodunnit` that shows the person who made the action. `set_paper_trail_whodunnit` setting identifies the current user and sets user id in `whodunnit`. 
![image](https://github.com/espoo-dev/med_system_backend/assets/92264330/124a8a53-bb5b-4036-8dca-6f09d452a65b)

Reply:
I see, my question is WDYF about adding it to both `ActionController::API` and `ApplicationController`

Reply:
Ohh, I got it! [Commit hash](https://github.com/espoo-dev/med_system_backend/pull/175/commits/47c34c303e09d0584de6bfd46f931c5921c0f5aa) `47c34c303e09d0584de6bfd46f931c5921c0f5aa`